maps/nerva/obj/nerva_machinery.dm

```diff
@@ -25,3 +25,6 @@
 	mask = /obj/item/clothing/mask/breath
 	req_access = list()
 	islocked = 1
+// Research Specific Air Alarms
+/obj/machinery/alarm/nervasci
+	req_access = list(list(access_xenobiology, access_atmospherics, access_engine_equip))
```

Comment:
This for me plus the area extending outside of isolation to trick the air alarm into covering those rooms in a no-go. If we're only doing this in 2 places (which I'll get around to in a moment), this doesn't need to be a new subtype define, just make a new instance in the map editor and set the vars there. This is unnecessary 

If we're doing this do give sci the ability to control the vents in the isolation room and such, then I'm personally against this approach. Rather than using the air alarms as a hacky way to do it, they should have an injector and be controlled remotely by an air control console.